### ISSUE TITLE
Add an option of using symlinks for Android's raw directory

### ISSUE THREAD

**Issue by sergeymorkovkin** [unimonkiez/react-native-asset#49]
Symlinks could eliminate file duplication in a project directory.

**Comment by unimonkiez** [comment]
Not sure I understood the issue, can you elaborate?

**Comment by sergeymorkovkin** [comment]
Resources get copied from their source directories to the respective Android's raw directory. Instead of copying these files (which are in our case are large enough), we could create sym-links at the destination path. I tried it on Android, and it worked.

**Comment by trippingtarballs** [comment]
Also not a fan of the copied artefacts. Duplicate files tracked under version control can cause confusion, and create opportunity for those files to get out of sync.

**Comment by unimonkiez** [comment]
Sounds like a good feature! will happily merge any PR in the subject

**Comment by sergeymorkovkin** [comment]
Yes, images get resized (so we have multiple-sized image files per one source image), but for sounds and fonts it's not justified at all... we're using symlinks already, and these work perfectly on MacOS.
